go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	contrib.go.opencensus.io/exporter/prometheus v0.1.1-0.20191218042359-6151c48ac7fa
 	github.com/cilium/ebpf v0.6.2
 	github.com/evanphx/json-patch v4.12.0+incompatible
+	github.com/gobwas/glob v0.2.3
```

Comment:
Can't we just use the stock go `filepath.Glob` implementation here ? It's probably less capable than this one, but if we allow multiple patterns and them as logically OR'd I think the stock one should be good enough (IIRC, the only thing it badly lacks is support for `foo{bar1,bar2,bar3}xyzzy`).